app/models/micropost.rb

```diff
@@ -0,0 +1,20 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  has_one_attached :image
+
+  PERMIT_ATTR = %i(content image).freeze
+  default_scope ->{order(created_at: :desc)}
+  validates :user_id, presence: true
```

Comment:
xóa dòng này đi đc ko em?

Reply:
xóa được ạ, vì micropost belong to :user thì đã có forein_key là user_id do vậy ko cần vẫn chạy được bình thường